packages/server/src/api/controllers/view/viewsV2.ts

```diff
@@ -0,0 +1,40 @@
+import sdk from "../../../sdk"
+import { Ctx, ViewV2 } from "@budibase/types"
+
+export async function fetch(ctx: Ctx) {
```

Comment:
Can we add the expected response type throughout the controller  i.e.`ctx: Ctx<void, ResponseType>`?
Ideally having `XRequest` and `XResponse` types, so we can decouple the web layer from the sdk